src/web/src/pages/CourseScheduler.vue

```diff
@@ -274,10 +274,25 @@ export default {
         } catch (err) {
           if (err.type == "Schedule Conflict") {
             if (i == course.sections.length - 1) {
+              //Compute Existing Course That Caused Conflict.
+              const existCourse = this.courses.find(function (currentCourse) {
+                //Search For Desired CRN In Current Course.
+                var isFound = currentCourse.sections.find(function (
+                  currentSection
+                ) {
+                  return currentSection.crn == err.existingSession.crn;
+                });
+                //Found Course w/ Appropriate CRN.
+                if (isFound) {
```

Comment:
an alternative/more semantic way of doing this is to use js array method `some` for determining if the current section has the `err.existingSession`